Source/hydro/riemann_nd.F90

```diff
@@ -1358,57 +1358,32 @@ subroutine riemannus(ql, ql_lo, ql_hi, &
              ! this just determines which of the left or right states is still
              ! in play.  We still need to look at the other wave to determine
              ! if the star state or this state is on the interface.
-
-             if (ustar > ZERO) then
-                ro = rl
-                uo = ul
-                po = pl
-                reo = rel
-                gamco = gamcl
-#ifdef RADIATION
-                lambda = laml
-                po_g = pl_g
-                po_r(:) = erl(:) * lambda(:)
-                reo_r(:) = erl(:)
-                reo_g = rel_g
-                gamco_g = gamcgl
-#endif
-
-             else if (ustar < ZERO) then
-                ro = rr
-                uo = ur
-                po = pr
-                reo = rer
-                gamco = gamcr
+             sgnm = sign(ONE, ustar)
```

Comment:
I don't think this works correctly if ustar == 0.

Reply:
you're right.  I added a fix there.